### DISCUSSION TITLE
How did you guys come up with the script?

### DISCUSSION THREAD

**Discussion by xkaku1** [pystardust/ani-cli#287] [Q&A] [answered]
What is the scripting process or process of creating this? How can a beginner like me make one for myself? How do you scrape?

**Comment by CoolnsX** [comment, marked as answer]
First analyse the website u want to scrape... Like how it's interacting.. how all the elements work.. then figure out ur use case.. like what u want to scrape..
 e.g. - u want to download wallpapers.. so u know and use the website.. so how to u use it like searching for wallpapers.. or just download any random wallpaper from the front page.. then learn the ``regex(regular expression)`` it is a way u find and match the pattern string.. like most of the website have uniform pages.. so regex will be the most useful technique for Scraping..
